docs/architecture/modernize-desktop/example-migration-core.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Example of migrating to .NET Core 3.1
+description: Showing how to migrate a sample applications targeting  .NET Framework to .NET Core 3.1.
+ms.date: 09/16/2019
+---
+
+# Example of migrating to .NET Core 3.1
+
+In this chapter, we present practical guidelines to help you perform a migration of your existing application from .NET Framework to .NET Core.
+
+We present a well-structured process you can follow and the most important things to consider on each step.
+
+We then document a step-by-step migration process for a sample desktop application, both from WinForms and WPF versions.
+
+## Migration process overview
+
+The migration process consists of four sequential steps:
+
+1. **Preparation**: Understand the dependencies the project has to have an idea of what's ahead. In this step, you take the current project into a state that simplifies the startup point for the migration.
+
+2. **Migrate Project File:** .NET Core projects use the new SDK-style project format. Create a new project file with this format or update the one you have to use the SDK style.
+
+3. **Fix code and build:** Build the code in .NET Core addressing API-level differences between .NET Framework and .NET Core. If needed, update third-party packages to the ones that support .NET Core. WCF clients.
```

Comment:
This WCF clients at the end of the paragraph seems lost?

Reply:
Indeed :)